python/cudnn/gemm/cutedsl/grouped/glu_hadamard_quant/api.py

```
@@ -85,8 +81,10 @@ def __init__(
         b_dtype: Optional[torch.dtype] = None,
         b_major: str = "k",
         sample_sfd: Optional[torch.Tensor] = None,
-        sample_rht: Optional[torch.Tensor] = None,
-        sample_sfrht: Optional[torch.Tensor] = None,
+        sample_rht_rowwise: Optional[torch.Tensor] = None,
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)